code/modules/arcade/mob_hunt/mob_datums.dm

```diff
@@ -0,0 +1,475 @@
+
+#define TYPE_FIRE /datum/mob_type/fire
+#define TYPE_WATER /datum/mob_type/water
+#define TYPE_GRASS /datum/mob_type/grass
+#define TYPE_ELECTRIC /datum/mob_type/electric
+#define TYPE_GROUND /datum/mob_type/ground
+#define TYPE_ROCK /datum/mob_type/rock
+#define TYPE_BUG /datum/mob_type/bug
+#define TYPE_POISON /datum/mob_type/poison
+#define TYPE_NORMAL /datum/mob_type/normal
+#define TYPE_FIGHTING /datum/mob_type/fighting
+#define TYPE_PSYCHIC /datum/mob_type/psychic
+#define TYPE_GHOST /datum/mob_type/ghost
+#define TYPE_ICE /datum/mob_type/ice
+#define TYPE_FLYING /datum/mob_type/flying
+#define TYPE_BLUESPACE /datum/mob_type/bluespace
+#define TYPE_DARK /datum/mob_type/dark
+#define TYPE_STEEL /datum/mob_type/steel
+
+/datum/mob_hunt
+	//GENERAL STATS AND VARIABLES
+	var/mob_name = "Generic Mob"			//the mob's original name (its species/type/whatever)
+	var/nickname = ""						//the mob's nickname (if given by the owner)
+	var/run_chance = 0						//percent chance the mob will escape capture attempts (higher is obviously more likely to get away)
+
+	//COMBAT STATS AND VARIABLES
+	var/level = 0							//actual level of this mob (don't set this, it gets overwritten in New())
+	var/min_level = 1						//minimum level of this mob (used for randomizing the actual level)
+	var/max_level = 1						//maximum level of this mob (used for randomizing the actual level)
+	//the types of the mob will be used for battles to determine damage resistance or weakness (mob_type_datums.dm)
+	var/datum/mob_type/primary_type			//Make sure you set this or the mob will be unable to deal damage and will take absurd damage in battles
+	var/datum/mob_type/secondary_type		//Don't set if not a dual-type mob so the mob will only calculate damage based on primary type
+	var/base_attack = 5						//base damage dealt by the mob's attacks for battling (effectively damage dealt at level 0)
+	var/base_health = 5						//base health of the mob for battling (effectively max health at level 0)
+	var/attack_multiplier = 1				//how much additional damage per level the mob deals (level * attack_multiplier)
+	var/health_multiplier = 1				//how much additional health per level the mob gets (level * health_multiplier) for calculating max health
+	var/health = list(0, 0)					//health[1] is the mob's current health, health[2] is the mob's max health (calculated in New())
```

Comment:
Storing both of these in a single list is terrible for readability, especially since you access it using magic numbers.